### DISCUSSION TITLE
Memory button for picture

### DISCUSSION THREAD

**Discussion by gentilsol** [balzack/databag#40] [Ideas]
I would be very happy if there was a button so you can speoch the photos

**Comment by balzack** [comment]
Can you describe this in more detail? I'm not sure the context of the button you are referring to.
Thanks.

**Reply by gentilsol** [reply]
With pleasure

If I send a picture or a photo in a topic, then the other person, the file can not save on his device.
Currently only a screenshot is possible

**Comment by balzack** [comment]
I see this works on iOS, I will look for a better solution on Android. If you tap on the message header, it presents you with a 'share' button. Clicking that will allow you to save to your device for iPhone, but for android it requires you to have a google photos account.

**Comment by geecrumb** [comment]
Just tried Databag and tried to download a picture on android. I second this: We need a "download" button next to the share button. Could be just a arrow pointing down.

**Comment by balzack** [comment]
I agree. I think I will be able to include this feature in the next mobile release. Thanks.

**Reply by geecrumb** [reply]
Thank you! This is why I have choosen databag as a new family chat, lol. I think you make great work and you put the effort into it to talk to your users.

**Comment by balzack** [comment]
Thank you! Talking with the users gives me a lot of energy and guidance.
